mediacenter/templates/mediacenter/index.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'two-third-third.html' %}
+{% load i18n %}
+
+{% block twothird %}
+    <h2><span class="theme read">{% trans "read" %}</span> {% trans "Medias Center" %}</h2>
+    <div class="grid document-list">
+        {% for document in document_list  %}
+            {% include "mediacenter/document_card.html" %}
```

Comment:
It's ok because you say you don't have a lot of date. But using `include` in a loop is a performance killer. Really. A lot.